tests/store/db/test_utils.py

```diff
@@ -21,9 +23,23 @@ def test_create_sqlalchemy_engine_inject_pool_options():
                 "mydb://host:port/",
                 pool_pre_ping=True,
                 pool_size=2,
-                pool_recycle=3600,
                 max_overflow=4,
+                pool_recycle=3600,
                 echo=True,
+                poolclass=NullPool,
+            )
+
+
+def test_create_sqlalchemy_engine_not_inject_pool_options():
+    with mock.patch.dict(
+        os.environ,
+        {},
+    ):
```

Comment:
```suggestion
def test_create_sqlalchemy_engine_not_inject_pool_options(monkeypatch):
    monkeypatch.delenv("MLFLOW_SQLALCHEMYSTORE_DISABLE_POOLING", raising=False)
```

We can use `monkeypatch.delenv` here.

Reply:
Never mind this comment. I thought this test verifies the `MLFLOW_SQLALCHEMYSTORE_DISABLE_POOLING` behavior.